app/javascript/packages/phone-input/index.ts

```diff
@@ -186,7 +186,7 @@ export class PhoneInputElement extends HTMLElement {
       return;
     }
 
-    const phoneNumber = textInput.value;
+    const phoneNumber = textInput.value as NationalNumber;
```

Comment:
I'm kinda confused by what they consider a "NationalNumber", since the value here may just be a partial fragment. But at least going by the [demo](https://catamphetamine.gitlab.io/libphonenumber-js/) and how it parses "National" under "As you type", it does seem like it's valid for it to just be the fragment, and just what we want when passing to `isValidNumberForRegion` and `isValidNumber`.

Otherwise, I would have thought that either `isValidNumberForRegion` and `isValidNumber` should be typed to accept any arbitrary string, or we could cast `phoneNumber` inline as the arguments of those functions.

tl;dr I think this is fine as-is..

Reply:
I think `NationalNumber` is a number without a country code, and our inputs don't have country code included, it matches